PR for Lesson02 Homework/Exercise (Grid Print)

LouReis opened the pull request:
This is a PR for the Grid Printing exercise from Lesson 2

Please see grid_part_final.py for the completed assignment.

natasha-aleksandrova commented:
@LouReis you closed this Pull Request - will you be resubmitting?


Please do _not_ add any assignments outside of your student directory.

LouReis commented:
Re-opened

natasha-aleksandrova commented:
Please remove the extra files per my earlier request.

LouReis commented:
Yes, I did
 

Sent: Tuesday, December 11, 2018 at 4:38 PM
From: Natasha <notifications@github.com>
To: UWPCE-PythonCert-ClassRepos/Self_Paced-Online <Self_Paced-Online@noreply.github.com>
Cc: LouReis <lou.reis@gmx.com>, "State change" <state_change@noreply.github.com>
Subject: Re: [UWPCE-PythonCert-ClassRepos/Self_Paced-Online] PR for Lesson02 Homework/Exercise (Grid Print) (#1237)


Please remove the extra files per my earlier request.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub, or mute the thread.

natasha-aleksandrova commented on students/LouReis/Lesson02/Fizz_Buzz.py:
```diff
@@ -0,0 +1,17 @@
+def fizz_buzz():
+    for x in range(0,101):
+        if x % 3 == 0 and x % 5 == 0:
+         print ("FizzBuzz")
```
this should have 4 spaces indents (everything is 4 spaces indent, no tabs)

LouReis commented:
Thanks for the feedback on the code.  Right, I haven't finished series.py yet.  I'll look into the indentation item...

 

Lou

 

Sent: Wednesday, December 12, 2018 at 9:31 AM
From: Natasha <notifications@github.com>
To: UWPCE-PythonCert-ClassRepos/Self_Paced-Online <Self_Paced-Online@noreply.github.com>
Cc: LouReis <lou.reis@gmx.com>, Mention <mention@noreply.github.com>
Subject: Re: [UWPCE-PythonCert-ClassRepos/Self_Paced-Online] PR for Lesson02 Homework/Exercise (Grid Print) (#1237)


@natasha-aleksandrova approved this pull request.

@LouReis code looks good, looks like series is not complete yet?

Please address the indentation issues, you should be able to configure it in your editor or use an IDE like Pycharm.


In students/LouReis/Lesson02/Fizz_Buzz.py:

> @@ -0,0 +1,17 @@
+# Fizz_Buzz.py
+# Fizz Buzz assignment
+# Coded by LouReis
+# Use % to see if there is a remainder from the division and compare with zero.
+
+def fizz_buzz():
+    for x in range(0,101):
+        if x % 3 == 0 and x % 5 == 0:
+         print ("FizzBuzz")


this should have 4 spaces indents (everything is 4 spaces indent, no tabs)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.

LouReis commented:
Hello,

 

I did a new pull request, here's the link:

 

https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/1245

 

The web site does not allow me to resubmit Exercise 1 of Lesson02 but there are 2 files that contain the "Grid Print" Exercise:

 

grid_print_final.py #I fixed the indentation and changed the name but the code is the same as grid_part_final.py which was submitted prior.

 

 

This new pull request should contain all files for all 3 exercises in Lesson02.  Lesson02 homework should all be completed, please let me know if that is not the case.

 

I submitted Lesson02 Exercise2 FizzBuzz through the submission page (PR 1245)

I submitted Lesson02 Exercise3 Fibonacci,Lucas,Sum Series through the submission page (PR 1245)

 

Please let me know if there's anything else I need to address, as of now I should have completed all homework/exercises for Lesson01 & Lesson02.

 

Thank you

Lou

 

Sent: Wednesday, December 12, 2018 at 9:31 AM
From: Natasha <notifications@github.com>
To: UWPCE-PythonCert-ClassRepos/Self_Paced-Online <Self_Paced-Online@noreply.github.com>
Cc: LouReis <lou.reis@gmx.com>, Mention <mention@noreply.github.com>
Subject: Re: [UWPCE-PythonCert-ClassRepos/Self_Paced-Online] PR for Lesson02 Homework/Exercise (Grid Print) (#1237)


Merged #1237 into master.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.
